src/test/java/net/snowflake/client/jdbc/SnowflakeResultSetSerializableIT.java

```diff
@@ -921,4 +928,21 @@ private boolean isMetadataConsistent(
         && actualTotalCompressedSize == expectedTotalCompressedSize
         && expectedTotalUncompressedSize == actualTotalUncompressedSize;
   }
+
+  @Test
+  @ConditionalIgnoreRule.ConditionalIgnore(condition = RunningOnGithubAction.class)
```

Comment:
Cannot this run in Github Action? Looks like this is a unit test that should be able to run in any environment?

Reply:
Yes, this case is unit test which doesn't depend on any env. I will remove the condition. (sorry, it is cloned from existing test.)